evoting/lib/election.go

```diff
@@ -118,23 +118,25 @@ func (e *Election) setVoted(s *skipchain.Service, user uint32) error {
 }
 
 func (e *Election) setStage(s *skipchain.Service) error {
-	db := s.GetDB()
-	latest, err := db.GetLatest(db.GetByID(e.ID))
+	target := 2 * len(e.Roster.List) / 3
```

Comment:
shouldn't this be configured during initialisation?

Reply:
sorry, didn't quite get you there. Did you mean during election initialization as a field in the `Election` struct?

Reply:
yeah, something like that. but this is just a suggestion. if you need to get these changes in soon, then please go ahead.

Reply:
Yes, sounds reasonable. At least open an issue for that (and hope we don't forget it ;)

Reply:
opened #1227